service/src/main/kotlin/fi/espoo/evaka/invoicing/data/VoucherValueDecisionQueries.kt

```diff
@@ -305,23 +306,34 @@ fun Database.Read.searchValueDecisions(
             VoucherValueDecisionSortParam.STATUS -> "decision.status"
         }
 
+    val retroactiveOnly =
+        distinctiveParams.contains(VoucherValueDecisionDistinctiveParams.RETROACTIVE)
+
     val params =
-        listOf(
+        listOfNotNull(
             Binding.of("page", page),
             Binding.of("pageSize", pageSize),
             Binding.of("status", status),
             Binding.of("areas", areas),
             Binding.of("unit", unit),
+            Binding.of("postOffice", postOffice),
             Binding.of("start_date", startDate),
             Binding.of("end_date", endDate),
             Binding.of("financeDecisionHandlerId", financeDecisionHandlerId),
             Binding.of("difference", difference),
-            Binding.of("firstPlacementStartDate", evakaClock.now().toLocalDate().withDayOfMonth(1))
+            Binding.of("firstPlacementStartDate", evakaClock.now().toLocalDate().withDayOfMonth(1)),
+            if (retroactiveOnly) Binding.of("now", evakaClock.now()) else null
```

Comment:
This binding doesn't have to be included conditionally, there are other bindings already that are not always used in the query.

Reply:
Yeah, should we add `service_need_missing` to `voucher_value_decision`-table?